web-build/Dockerfile.task

```
@@ -0,0 +1,4 @@
+#ddev-generated
+COPY install-task.sh /usr/local/bin
```

Hadolint (2.12.0)

[error] Invalid instruction order. Dockerfile must begin with `FROM`, `ARG` or comment.

(DL3061)